src/runmode-netmap.c

```diff
@@ -149,7 +149,9 @@ static int ParseNetmapSettings(NetmapIfaceSettings *ns, const char *iface,
             ns->threads = 0;
             ns->threads_auto = true;
         } else {
-            ns->threads = atoi(threadsstr);
+            if (ByteExtractStringUint16(&ns->threads, 10, 0, (const char *)threadsstr) < 0) {
+                goto finalize;
```

Comment:
1. Suggest a message stating the configured value is invalid.
2. Not sure if jumping to `finalize` is correct.
3. `ns->threads` is indeterminate. Is a default value appropriate here?

Reply:
> 1. Suggest a message stating the configured value is invalid.
OK
> 2. Not sure if jumping to `finalize` is correct.
> 3. `ns->threads` is indeterminate. Is a default value appropriate here?
all other cases use `0`, does it make sense to use that here as well. These conditional checks (L144-L158 in this branch) won't really make sense if we set the same value in every block.